src/internal.rs

```diff
@@ -783,6 +804,26 @@ pub(crate) struct Rav1dFrameContext_task_thread {
     pub pending_tasks: Mutex<Rav1dFrameContext_task_thread_pending_tasks>,
 }
 
+impl Default for Rav1dFrameContext_task_thread {
+    fn default() -> Self {
+        Self {
+            lock: Default::default(),
+            cond: Default::default(),
+            ttd: Default::default(),
+            tasks: Default::default(),
+            init_done: Default::default(),
+            done: Default::default(),
+            retval: Mutex::new(Ok(())),
```

Comment:
nit just cause this `impl` feels annoying: we could make `retval` an `Option<Rav1dError>`.  Fine for now, though.

Reply:
Yeah, but not important for now.